embeds/linux_icon.go

```diff
@@ -0,0 +1,62 @@
+package embeds
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/anywherelan/awl/config"
+)
+
+const (
+	awlDirPath  = ".local/share/Anywherelan"
+	awlIconName = "Anywherelan.png"
+
+	dirMod  = 0700
+	iconMod = 0664
+)
+
+func EmbedIcon(iconBytes []byte) (string, error) {
+	homeDir := os.Getenv("HOME")
+	if len(homeDir) == 0 {
+		return "", errors.New("error: empty home dir")
+	}
+
+	dirPath := filepath.Join(homeDir, awlDirPath)
+	err := os.Mkdir(dirPath, dirMod)
+	if err != nil && !os.IsExist(err) {
+		return "", fmt.Errorf("error: create dir: %w", err)
+	}
+	err = os.Chown(dirPath, config.LinuxFilesOwnerUID, config.LinuxFilesOwnerUID)
+	if err != nil {
+		return "", fmt.Errorf("error: chown dir: %w", err)
+	}
+
+	iconPath := filepath.Join(dirPath, awlIconName)
+	file, err := os.Create(iconPath)
```

Comment:
I have one concern here: we already write icon to filesystem in `cmd/awl-tray/main.go` with this line: `_ = os.WriteFile(tempIconFilepath, appIcon, 0666)` for all platforms. I think it'd be great to have one common path with writing icons for all platforms.

Also, we can think about using `dataDir` from Config struct

Reply:
I thought we would use `os.UserConfigDir()/Anywherelan/` dir for icon on windows (and so we don't have to delete windows icon on app shutdown), but this way is OK too

Reply:
ok, it wasn't clear for me. 

And still I have some questions) 

Do you want me to use os.UserConfigDir() (AppData) or AWL_DATA_DIR if it is set as well?

Reply:
I do it with ```os.UserConfigDir()```, give me a sign if it is not ok

Reply:
It could be `AWL_DATA_DIR`, yes, I have a concern that it's rather dynamic variable and we will pollute filesystem with many icons

But `os.UserConfigDir()` is rather static, so as icon itself, so I thinks it's a little more preferable (but both options are OK I think)